### PR TITLE
ISPN-16533 Increase the socket timeout logging time

### PR DESCRIPTION
**Pull request by fax4ever**
https://issues.redhat.com/browse/ISPN-16533

@tristantarrant I increased the socket timeout to 5 min, logging the actual time.
The output of the log is not present on the Maven process log, but on the `target/infinispan-infinispan-client-hotrod.log`.

On my machine for instance, I have:
```
11:34:31,517 INFO  (testng-LargePutAllPressureTest:[]) [o.i.c.h.AuthenticationTest] test executed in 9776ms
```

### CODE REVIEW

**Comment by fax4ever** [comment]
> Let's just make the timeout 1 minute and see if it happens again

Sure! About the other issue, I'm working on it

**Comment by fax4ever** [comment]
1 min seems to be not enough... shall I resume 5 or maybe at this point it is better to recover the other PR (https://github.com/infinispan/infinispan/pull/12869). For me it is the same. Thanks for help

**Comment by tristantarrant** [comment]
Just leave it at 5 minutes

**Comment by fax4ever** [comment]
@tristantarrant is OK for you those changes?

**Comment by fax4ever** [comment]
I added the same changes to the `hotrod-new` module

File: client/hotrod-client-new/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java [hunk 1]
```diff
@@ -9,19 +9,29 @@
...
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "org.infinispan.client.hotrod.query.pressure.LargePutAllPressureTest")
 public class LargePutAllPressureTest extends SingleHotRodServerTest {
 
-   private final static int SIZE = 15_000;
+   private static final Log log = LogFactory.getLog(AuthenticationTest.class);
+
+   private static final int SIZE = 15_000;
+   private static final int TIMEOUT = 1_000 * 60; // 1 minute
```
Comment (wburns) [inline client/hotrod-client-new/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java:34]: I would recommend doing TimeUnit.MINUTES.toMillis(1) in the future. But I won't have that stop merging.

**Review by wburns — APPROVED** [review]

File: client/hotrod-client-new/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java [hunk 2]
```diff
@@ -9,19 +9,29 @@
...
 import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "org.infinispan.client.hotrod.query.pressure.LargePutAllPressureTest")
 public class LargePutAllPressureTest extends SingleHotRodServerTest {
 
-   private final static int SIZE = 15_000;
+   private static final Log log = LogFactory.getLog(AuthenticationTest.class);
```
Comment (wburns) [inline client/hotrod-client-new/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java:31]: Wrong class name :)

File: client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java [hunk 3]
```diff
@@ -9,19 +9,29 @@
...
 import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.util.logging.Log;
+import org.infinispan.util.logging.LogFactory;
 import org.testng.annotations.Test;
 
 @Test(groups = "functional", testName = "org.infinispan.client.hotrod.query.pressure.LargePutAllPressureTest")
 public class LargePutAllPressureTest extends SingleHotRodServerTest {
 
-   private final static int SIZE = 15_000;
+   private static final Log log = LogFactory.getLog(AuthenticationTest.class);
```
Comment (wburns) [inline client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java:31]: Same here about the name.

**Review by wburns — COMMENTED** [review]
LGTM, just a minor issue

**Comment by fax4ever** [comment]
Thank @wburns. I think I've applied the changes here.

**Comment by wburns** [comment]
Integrated into main, thanks @fax4ever !

**Comment by fax4ever** [comment]
Thanks @wburns

**Pull request merged**